app/experimenter/nimbus-ui/src/components/Summary/index.test.tsx

```diff
@@ -351,7 +351,7 @@ describe("Summary", () => {
     await act(async () => void fireEvent.click(requestUpdateButton));
   });
 
-  it("shows update and end buttons for live dirty rollout", async () => {
+  it("shows update and end experiment button for live dirty rollout", async () => {
```

Comment:
don't you mean don't show end experiment button?

Reply:
nope! it **should show** end experiment and it **should not** show end enrollment

Reply:
I could separate that out into another test 🤔

Reply:
this test could be "show these two buttons" and a new one could be "don't show this button" 🤔

Reply:
right, yeah that's what I thought, test case means show end enrollment but it wasn't doing that :D